include/utilities/parallel_utils.h

```diff
@@ -148,6 +148,15 @@ namespace parallel {
         // Start with a value of true
         bool isGood = true;
 
+        #ifndef ACTIVATE_PYTHON
```

Comment:
Put something similar in #303.  Will note you can reuse the `mpiSynStatsAnd` if you flip the `ifndef` for `ifdef`.  See [this patch](https://github.com/NOAA-OWP/ngen/pull/303/files#diff-af36840b7b7237a03bc3ffcb57bea59687b7ce5211ad1b393f188c9819cdf018) if you are interested.